contracts/static_analysis.sol

```diff
@@ -1,55 +1,55 @@
 // SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
This file is generated for static analysis.. Feel free to delete it if it is interfering in dev / test..